src/pages/NewsPage/index.tsx

```diff
@@ -0,0 +1,104 @@
+//Assets
+import React from 'react';
+import { IoIosArrowBack } from 'react-icons/io';
+import newsThumbnail from '@/assets/newsThumbnail@3x.png';
+
+const Button = ({
+  title,
+  thumbnailImage,
+  imageClassName,
+}: {
+  title: string;
+  thumbnailImage?: string;
+  imageClassName?: string;
+}) => {
+  return (
+      <button className='h-14 w-48 rounded-[1.5rem] text-xl text-red-500 bg-white border border-red-500 hover:font-bold hover:border-red-500 hover:border-2 px-4'>

```

Comment:
if you can combine this button with the button for the other page, and make it a common component under `src/components`, that would be awesome to have a reusable component!

Reply:
I think try use this method:https://stackoverflow.com/questions/59048346/extending-component-props-to-default-html-button-props-correctly 
This allows us to add attributes as props into the button component